src/create_disk.sh

```diff
@@ -317,7 +317,14 @@ if [ -e (md/md-boot) ]; then
   # the main grub.cfg.
   set prefix=md/md-boot
 else
-  search --label boot --set prefix --no-floppy
+  if [ -f ${config_directory}/boot_uuid ]; then
+    source ${config_directory}/boot_uuid
```

Comment:
In the UEFI case we're putting a non-8.3 filename on a FAT filesystem.  That's probably fine, but should we rename the file anyway?

Reply:
Makes sense to me. How about `bootuuid.cfg`?

Reply:
WFM!